application-analytics-ui/src/main/resources/Analytics/Deafult Dashbord/WebHome.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="Analytics.Deafult Dashbord.WebHome" locale="">
+  <web>Analytics.Deafult Dashbord</web>
+  <name>WebHome</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1689592941000</creationDate>
+  <parent>Analytics.WebHome</parent>
+  <author>XWiki.Admin</author>
+  <contentAuthor>XWiki.Admin</contentAuthor>
+  <date>1689592944000</date>
+  <contentUpdateDate>1689592941000</contentUpdateDate>
+  <version>2.1</version>
+  <title>Deafult Dashbord</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content/>
+  <object>
+    <name>Analytics.Deafult Dashbord.WebHome</name>
```

Comment:
You have a typo in the name, and you could also use PascalCase, as for the other pages (just without the space in the name)   `DefaultDashboard`
Also do you need it to be non-terminal age? I think you could mention from the template provider that terminal pages sould be created and let users modify it when they don't want it.